addons/search-sprites/userscript.js

```diff
@@ -17,28 +26,63 @@ export default async function ({ addon, global, console, msg }) {
   // the libraries, so this fits right in.
   searchBox.type = "text";
 
-  const search = (query) => {
+  // These functions were taken from the search sprites addon
+  const getFolderFromName = (name) => {
+    const idx = name.indexOf("//");
+    if (idx === -1 || idx === 0) {
+      return null;
+    }
+    return name.substr(0, idx);
+  };
+
+  const getNameWithoutFolder = (name) => {
+    const idx = name.indexOf("//");
+    if (idx === -1 || idx === 0) {
+      return name;
+    }
+    return name.substr(idx + DIVIDER.length);
+  };
+
+  const search = (query, sprites) => {
     if (!spritesContainer) return;
 
     query = query.toLowerCase();
     const containsQuery = (str) => str.toLowerCase().includes(query);
 
+    console.log(typeof sprites);
+    console.log(sprites);
```

Comment:
Very nice! But maybe we don't want to send this and line above to console? It is messy now, so we don't want to make it worse... But this thing `Matching folders...` can stay.

Reply:
Thanks! About the `console.log()`s, I thought removed them with commit 1c1d62c30132923006659d98af318b888f456317. Can you check again?

Reply:
Sorry. I didn't saw this commit.. So NP